src/Service/TraceContext/TraceContextRenderer.php

```diff
@@ -0,0 +1,30 @@
+<?php
+declare(strict_types=1);
+
+namespace DR\SymfonyTraceBundle\Service\TraceContext;
+
+use DR\SymfonyTraceBundle\TraceContext;
+
+class TraceContextRenderer
+{
+    public static function renderTraceParent(TraceContext $trace): string
```

Comment:
Created separated renderer to allow rendering the TraceContext to string outside the library.